app/src/app/map/Map.tsx

```diff
@@ -1,5 +1,42 @@
+import React from "react";
+
 import { MapView } from "ui/map/map-view/MapView";
+import { WalletSelectorNavbar } from "ui/wallet-selector-navbar/WalletSelectorNavbar";
+import { WalletSelector } from "ui/wallet-selector/WalletSelector";
+import { MapMarker } from "ui/map/map-marker/MapMarker";
+
+import { MapProps, PropertyMapMarker } from "./Map.types";
+import styles from "./Map.module.scss";
+
+export const Map: React.FC<MapProps> = ({ properties }) => {
+  const [location, setLocation] = React.useState({ lat: 14.62, lng: -90.53 });
+
+  const setUserLocation = (position: GeolocationPosition) => {
+    const {
+      coords: { longitude: lng, latitude: lat },
+    } = position;
+
+    setLocation({ lat, lng });
+  };
 
-import { MapProps } from "./Map.types";
+  React.useEffect(() => {
+    if ("geolocation" in navigator) {
+      navigator.geolocation.watchPosition(setUserLocation);
+    }
+  }, []);
 
-export const Map: React.FC<MapProps> = ({ mapOptions }) => <MapView mapOptions={mapOptions} />;
+  return (
+    <>
+      <WalletSelectorNavbar>
+        <WalletSelector />
+      </WalletSelectorNavbar>
```

Comment:
Pull `master`. This has been updated to be: 

```suggestion
      <WalletSelectorNavbar />
```